Xkcd-Comics/ContentViews/ContentView.swift

```diff
@@ -9,60 +9,64 @@ import SwiftUI
 import Kingfisher
 
 struct ContentView: View {
-    @StateObject var comicsVM = ComicsVM()
+    @StateObject var viewModel: ComicsVM
```

Comment:
Injected dependencies in SwiftUI views should be `@ObservedObject`  

> There is one important difference between @StateObject and @ObservedObject, which is ownership – which view created the object, and which view is just watching it.

https://www.hackingwithswift.com/quick-start/swiftui/whats-the-difference-between-observedobject-state-and-environmentobject